src/components/map/map-header.tsx

```diff
@@ -59,13 +64,13 @@ export function MapHeader({ id, name, layer, transportModes }: MapHeaderProps) {
         <div className={style.header__buttons}>
           <ButtonLink
             mode="interactive_0"
-            href={`/planner?travelFrom=${id}`}
+            href={`/assistant?fromLat=${position.lat}&fromLon=${position.lon}&fromLayer=${layer}`}
```

Comment:
I think this could also be URL and maybe easier to read?

```
            href={{ paths: '/assistant', query: { your object here} }}
```


IF i added that to the ButtonLink back in the day 🤔

Reply:
Probably. Can look into it if we want it, but have to be later.